libdenaro/src/models/dashboardgroup.cpp

```diff
@@ -0,0 +1,25 @@
+#include "models/dashboardgroup.h"
+
+namespace Nickvision::Money::Shared::Models
+{
+    DashboardGroup::DashboardGroup()
+        : m_data{ }
+    {
+    }
+
+    const std::unordered_map<Nickvision::Money::Shared::Models::Currency, std::pair<double, std::string>>& DashboardGroup::getData() const
+    {
+        return m_data;
+    }
+
+    void DashboardGroup::addAmount(const Nickvision::Money::Shared::Models::Currency& currency, double amount, const std::string& accountName)
+    {
+        if(!m_data.contains(currency))
+        {
+            m_data.emplace(currency, std::make_pair(amount, currency.getSymbol() + " " + std::to_string(amount) + " from " + accountName));
```

Comment:
The account string needs to be translated:

Add the following includes to the beginning of the file:
```c
#include <format>
#include <libnick/localization/gettext.h>
#include "helpers/currencyhelpers.h"
```

then replace the string in `make_pair` with `std::vformat(_("{} from {}"), std::make_format_args(CurrencyHelpers::toAmountString(amount, currency), accountName));`

You also need to add this file path to `/resources/po/POTFILES` that way it gets picked up in the po files.